features/dd-sdk-android-trace/lint-baseline.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <issue id="NewApi">
+        <ignore path="src/main/java/com/datadog/trace"/> />
```

Comment:
I guess this is temporary? I thought the idea is to have `dd-sdk-android-trace` completely compatible with API 21 as it was before?

Reply:
nope it is not temporary and it cannot be compatible as long as it uses the `java.util.function` package. The `CoreTracer` code needs to stay in the `dd-sdk-android-trace` and for now it will be used only by the `dd-sdk-android-trace-otel` module which is going to require desugaring in the docs and also which is protected in case someone omits the docs.

Reply:
The idea is not to block dogfooding too much. Maybe while dogfooding I can clean that code a bit more and find a way to make it compatible. I doubt it though as I tried at some point and it seems impossible without replacing that package with a `java.util.function.adapter` package, they are being used in too many places.